tests/results/SourceRangeTest.py

```diff
@@ -86,6 +86,31 @@ def test_json(self):
                                       end_line=4).__json__(use_relpath=True)
         self.assertEqual(uut['start'], self.result_fileB_line2)
 
+    def test_contains(self):
+        a = SourceRange(SourcePosition("test_file", 1, 2),
+                        SourcePosition("test_file", 1, 20))
+        b = SourceRange(SourcePosition("test_file", 1, 1),
+                        SourcePosition("test_file", 1, 20))
+        self.assertIn(a, b)
+
+        a = SourceRange(SourcePosition("test_file", 1, 2),
+                        SourcePosition("test_file", 2, 20))
+        b = SourceRange(SourcePosition("test_file", 1, 1),
+                        SourcePosition("test_file", 2, 20))
+        self.assertIn(a, b)
+
+        a = SourceRange(SourcePosition("test_file", 1, 2),
```

Comment:
We have an easier way of doing this using `SourceRange.from_values()`
Would make the test easier to read